addons/sale_stock/models/sale_order.py

```diff
@@ -235,15 +235,15 @@ def _compute_qty_at_date(self):
         grouped_lines = defaultdict(lambda: self.env['sale.order.line'])
         # We first loop over the SO lines to group them by warehouse and schedule
         # date in order to batch the read of the quantities computed field.
+        now = fields.Datetime.now()
         for line in self:
             if not line.display_qty_widget:
                 continue
             line.warehouse_id = line.order_id.warehouse_id
             if line.order_id.commitment_date:
                 date = line.order_id.commitment_date
             else:
-                confirm_date = line.order_id.date_order if line.order_id.state in ['sale', 'done'] else datetime.now()
```

Comment:
After consideration, still useful for new lines in confirmed SO's, to verify and bring back